draft-packages/form/KaizenDraft/Form/CheckboxField/styles.scss

```diff
@@ -32,15 +32,6 @@
     margin-bottom: 0;
   }
 
-  @media (hover: hover) and (pointer: fine) {
```

Comment:
I think this style was redundant but could be wrong.

Reply:
Hey just did a dig, mmiiiight worth keeping after reading this commit message, but see what you think. It fixes some iOS issues apparently.
https://github.com/cultureamp/kaizen-design-system/commit/da66a828aea401187ae99f7e6fa55975fce8b7a7#diff-13e9c7a2fe4b0fbfb5b65f6566c7794bf2bf639ac0c6c8bbf7414c5cdba168db

Reply:
Thanks for checking! I've reverted.